src/app/jobs/weatherevents.job.ts

```diff
@@ -17,67 +18,31 @@ export class WeatherEventsJob extends Job {
     const resp = await container.httpService.get("https://api.weather.gov/alerts/active?area=FL");
     const data: Weather_Event = resp.data;
 
-    if (!data.features) {
+    if (!data.features || data.features.length == 0) {
       return;
     }
 
-    const map: string[] = data.features.map(f => f.properties.headline);
+    const relevantData: FeaturesEntity[] = data.features.filter(rD => {
+      const highSeverity = rD.properties.severity === 'Severe' 
+                          || rD.properties.severity === 'Extreme';
 
-    if (map.length == 0) {
-      return;
-    }
+      return highSeverity && rD.properties.headline.includes("Orlando")

```

Comment:
Is there not a field that tells the location, rather than looking through the headline?

Reply:
After taking a second look through some JSONs, senderName seems to help with that too. It usually says something like NWS <city>. In all of the headlines I've seen, the sendername is always mentioned in the headline too, but for correctness sake, I'll use the senderName instead since that makes more sense.

Reply:
This is what it looks like for one ping to the API. Here, you can see the issue that I described below about duplicate headlines and what not.
![image](https://user-images.githubusercontent.com/49847280/123537938-26736280-d700-11eb-8fd8-7ec0bb636551.png)